changestopiglatin.txt

```diff
@@ -0,0 +1,13 @@
+
+pyg = 'ay'
+
+original = raw_input('Enter a word:')
+
+if len(original) > 0 and original.isalpha():
+    word = original.lower()
+    first = word[0]
+    new_word = word+first+pyg
+    new_word = new_word[1:len(new_word)]
```

Comment:
While possible to bind `new_word` to the last version of `new_word` with a mutation, this is highly discouraged.  The more different things you call the same name, the more confusing your code gets.  Try to always avoid reusing the same variable name for different things.  If the same variable will be passed to different functions, then you SHOULD call it the same thing, but never call different things the same variable name.